CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### 🔧 Internal changes
 
+## [2.11.2] - 2025-09-12
```

Comment:
You should not be adding a new release section; instead, add this entry just under the unreleased section